sdk/storage/Azure.Storage.Common/tests/PooledMemoryStreamTests.cs

```diff
@@ -117,6 +117,7 @@ public void ReadStream(int dataSize, int bufferPartitionSize)
 
         [Test]
         [Explicit]
+        [Ignore("Makes running unit tests locally extremely slow")]
```

Comment:
how long does this take?

Reply:
Several minutes.  I run the unit tests multiple times as part of my workflow, so it adds up quickly.